client.js

```diff
@@ -162,14 +164,24 @@ function streamAuction (auction_user, auction_contents, server) {
     let channelID;
 
     if (server === "GREEN") {
-        channelID = "672512233435168784";
+        // channelID = "672512233435168784";
+        //for test server user...
+        channelID = "716521067388338207";
     }
-    // console.log(msg, server, channelID);
+    // console.log("bot.channels.get", bot.channels.fetch("716521067388338207"));
 
     auction_contents = auction_contents.replace(/[|]+/g, '|');
     fetchAndFormatAuctionData(auction_user, auction_contents, server).then(formattedAuctionMessage => {
-        // in the real code this would do:
-        bot.channels.cache.get(channelID).send(formattedAuctionMessage)
+
+        bot.channels.fetch("716521067388338207")
```

Comment:
Need to actually USE `channelID` :D

Reply:
channelID.toString()

I still don't know why.  Maybe a quirk of Discord.js.